### PR TITLE
Add pages-plugin-trpc into Community Pages Plugins

### PR DESCRIPTION
**Pull request by toyamarinyon**
Hi 👋 

I really exited for many new service that announced on Cloudflare Platform week.

Especially, I think that SQLite on Edge called D1 and Cloudflare Pages Plugin quite change web development.

By the way, do you know tRPC? tRPC is library that allows End-to-end typesafe APIs made easy. 

So, I’am considering that D1 and tRPC allows  End-to-end typesafe application made quite easy and cost-effective.

Therefore I decided to try it out as soon as D1 was released, and first created a page plugin for tRPC.

### CODE REVIEW

File: content/pages/platform/functions/plugins/community-plugins.md [hunk 1]
```diff
@@ -13,6 +13,11 @@ The following are some of the community-maintained Pages Plugins. If you have cr
 
   Given a folder of assets in multiple formats, this Plugin will automatically negotiate with a client to serve an optimized version of a requested asset.
 
+
+- [pages-plugin-trpc](https://github.com/toyamarinyon/cloudflare-pages-plugin-trpc)
+
+    It allows developers to create tRPC server on Cloudflare Page Function rapidly.
```
Comment (deadlypants1973) [inline content/pages/platform/functions/plugins/community-plugins.md]:
```suggestion
    Allows developers to quickly create a tRPC server with a Cloudflare Pages Function.
```

**Comment by kristianfreeman** [comment]
cc @GregBrimble @Ekwuno

File: content/pages/platform/functions/plugins/community-plugins.md (hunk 1 above)
Comment (toyamarinyon) [reply content/pages/platform/functions/plugins/community-plugins.md]: Thank you for rewriting.

File: content/pages/platform/functions/plugins/community-plugins.md [hunk 2]
```diff
@@ -13,6 +13,11 @@ The following are some of the community-maintained Pages Plugins. If you have cr
 
   Given a folder of assets in multiple formats, this Plugin will automatically negotiate with a client to serve an optimized version of a requested asset.
 
+
+- [pages-plugin-trpc](https://github.com/toyamarinyon/cloudflare-pages-plugin-trpc)
```
Comment (GregBrimble) [inline content/pages/platform/functions/plugins/community-plugins.md]:
```suggestion
- [cloudflare-pages-plugin-trpc](https://github.com/toyamarinyon/cloudflare-pages-plugin-trpc)
```

Looks like [it's published to npm](https://www.npmjs.com/package/cloudflare-pages-plugin-trpc) under the name `cloudflare-pages-plugin-trpc`.

Comment (GregBrimble) [reply content/pages/platform/functions/plugins/community-plugins.md]: Other than that (and potentially now re-ordering in the list), LGTM!

Comment (toyamarinyon) [reply content/pages/platform/functions/plugins/community-plugins.md]: Thank you for rewriting!

**Review by GregBrimble — APPROVED** [review]

**Review by deadlypants1973 — APPROVED** [review]

**Comment by toyamarinyon** [comment]
thanks for merging 😄

**Pull request merged**